File: OpenApi2.0/push-openapi-php-demo/PushNoticeToiOS.php
<?php

include_once '../aliyun-php-sdk-core/Config.php';
include 'Push/Request/V20160801/PushNoticeToiOSRequest.php';
use \Push\Request\V20160801 as Push;

// 设置你的AccessKeyId/AccessSecret/AppKey

$accessKeyId = "your accessKeyId";
$accessKeySecret = "your accessKeySecret";
$appKey = "your appKey";

$iClientProfile = DefaultProfile::getProfile("cn-hangzhou", $accessKeyId, $accessKeySecret);
$client = new DefaultAcsClient($iClientProfile);
$request = new Push\PushNoticeToiOSRequest();


$request->setAppKey($appKey);
$request->setTarget("ALL"); //推送目标: DEVICE:推送给设备; ACCOUNT:推送给指定帐号,TAG:推送给自定义标签; ALL: 推送给全部
$request->setTargetValue("ALL"); //根据Target来设定，如Target=DEVICE, 则对应的值为 设备id1,设备id2. 多个值使用逗号分隔.(帐号与设备有一次最多100个的限制)
$request->setApnsEnv("DEV");//iOS的通知是通过APNS中心来发送的，需要填写对应的环境信息. DEV :表示开发环境, PRODUCT: 表示生产环境
$request->setBody("php Body");
$request->setExtParameters("{\"k1\":\"v1\",\"k2\":\"v2\"}");


$response = $client->getAcsResponse($request);
print_r("\r\n");
print_r($response);

?>

File: OpenApi2.0/push-openapi-net-demo/PushNoticeToiOS.cs
using Aliyun.Acs.Core;
using Aliyun.Acs.Core.Exceptions;
using Aliyun.Acs.Core.Profile;
using Aliyun.Acs.Push.Model.V20160801;
using System;

namespace AlibabaCloud
{
    class Push
    {
        static void Main()
        {
            IClientProfile clientProfile = DefaultProfile.GetProfile("cn-hangzhou", "<your access key id>", "<your access key secret>");
	    DefaultAcsClient client = new DefaultAcsClient(clientProfile);
            PushNoticeToiOSRequest request = new PushNoticeToiOSRequest();
            request.AppKey = <your appKey>;
            // iOS的通知是通过APNS中心来发送的，需要填写对应的环境信息. DEV:表示开发环境, PRODUCT: 表示生产环境
            request.ApnsEnv = "DEV";
            request.Target = "ALL";
            request.TargetValue = "all";
            request.Title = "iOS Notice Title";
	    request.Body = "iOS Notice Body";
            request.ExtParameters = "{\"k1\":\"v1\",\"k2\":\"v2\"}";
            try
            {
                PushNoticeToiOSResponse response = client.GetAcsResponse(request);
                Console.WriteLine("RequestId:" + response.RequestId);
                Console.WriteLine("ResponseId:"+response.MessageId);
                Console.ReadLine();
            }
            catch (ServerException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
		Console.ReadLine();
            }
            catch (ClientException e)
            {
                Console.WriteLine(e.ErrorCode);
                Console.WriteLine(e.ErrorMessage);
		Console.ReadLine();
            }
        }
            
    }
}

